### DISCUSSION TITLE
Any plan to add Metatrader 5?

### DISCUSSION THREAD

**Discussion by Qoyyuum** [blankly-finance/blankly#186] [Ideas]
Is there any plan to add Metatrader 5 since it has Python support? https://www.mql5.com/en/docs/integration/python_metatrader5

**Comment by bfan1256** [comment]
Hey @Qoyyuum , at this moment we don't have a plan to add MT5, we've had a lot of interest for IBKR and futures work, so those have been prioritized a little bit more. Happy to hear more though if you'd like to reach out to us: hello@blankly.finance :D

**Reply by Qoyyuum** [reply]
Oh that's Ok. Once I have time, may I add MT5 to blankly?

**Reply by EmersonDove** [reply]
Sure, we're happy to accept contributions. Feel free to join our discord, we answer a lot of questions there.

**Reply by guzuomuse** [reply]
game changer if we have mt5 supported!!!
